Sources/SwiftKuery/Table.swift

```diff
@@ -82,8 +106,22 @@ open class Table: Buildable {
         }
     }
 
-    /// Build the table using `QueryBuilder`.
-    ///
+    // MARK: String Representation
+    /**
+     Function to build a String representation for referencing a `Table` instance.
+     A `QueryBuilder` is used handle variances between the various database engines and produce a correct SQL description.
```

Comment:
"...produce a correct SQL description for the database engine which is being used." ?